Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.messages.in

```diff
@@ -27,6 +27,7 @@
 
 #if ENABLE(GPU_PROCESS) && HAVE(AVASSETREADER)
 
+[EnabledBy=MediaPlaybackEnabled]
```

Comment:
Can we use `UseGPUProcessForMediaEnabled` instead?
It would be more accurate I think.
Maybe we can change `UnifedWebPreferences.yaml` to mark  `UseGPUProcessForMediaEnabled`  as `sharedPreferenceForWebProcess: true`?